x-pack/plugins/fleet/server/services/epm/packages/bulk_install_packages.ts

```diff
@@ -35,16 +34,13 @@ export async function bulkInstallPackages({
 }: BulkInstallPackagesParams): Promise<BulkInstallResponse[]> {
   const logger = appContextService.getLogger();
 
-  // auto upgrade to prerelease versions only if the setting is enabled
-  const prerelease = await getPrereleaseFromSettings(savedObjectsClient);
```

Comment:
removed using the settings after discussing here https://github.com/elastic/kibana/issues/122973#issuecomment-1297047329